contracts/RSTCrowdsale.sol

```diff
@@ -29,4 +28,4 @@ contract RSTCrowdsale is CappedCrowdsale, RefundableCrowdsale, MintedCrowdsale {
     
   }
 
-}
+}
```

Comment:
Restore the changes on this file

Reply:
Also, this contract should be upgradeable too but looks like you missed it.

Reply:
@ikuteer , I'm sorry I should have clarified. This crowdsale contract is not upgradable using any proxy pattern. One of the contract that `RSTCrowdsale` inherits is `RefundableCrowdsale`. This contract has a private variable `escrow` which cannot be set using the proxy pattern architecture. I even consulted the zeppelin os developers to ask about this. Here's a screeshot of my interaction - 
![screen shot 2018-09-02 at 3 26 18 pm](https://user-images.githubusercontent.com/20184946/44954739-9f0c3300-aec4-11e8-9f10-5b7c1139828e.png)

Since it might not be safe to tamper the internal workings of the crowdsale architecture & 
 `RSTCrowdsale` contract will only be ever used once during the crowdsale, I would recommend not trying to make it upgradable.

The current setup is like this- you deploy a `ResearchSpecificToken` using proxy pattern, this will give you a proxy address to this token, you can then use this proxy token address inside the original `RSTCrowdsale` contract.